pkg/plugins/manager/client/client.go

```diff
@@ -72,6 +72,10 @@ func (s *Service) QueryData(ctx context.Context, req *backend.QueryDataRequest)
 			return nil, err
 		}
 
+		if errors.Is(err, context.Canceled) {
+			return nil, plugins.ErrPluginRequestCanceledErrorBase.Errorf("client: query data request cancelled: %w", err)
```

Comment:
Nit for consistency with variable name:
```suggestion
			return nil, plugins.ErrPluginRequestCanceledErrorBase.Errorf("client: query data request canceled: %w", err)
```